integration/python/wrapper_collection.cpp

```diff
@@ -64,7 +64,9 @@ namespace ottergon {
             auto doc = to_document(document);
             generate_document_id_if_not_exists(doc);
             auto session_tmp = ottergon::session_id_t();
-            auto result = ptr_->insert_one(session_tmp, database_, name_, doc);
+            auto result_variant = ptr_->insert_one(session_tmp, database_, name_, doc);
+            assert(result_variant.is_type<components::result::result_insert>() && "wrapper_collection::insert_one result error");
```

Comment:
Possible `error_result_t`

Reply:
How I should process it in this case? raise exception?